달력새거.html

```diff
@@ -0,0 +1,46 @@
+<!DOCTYPE html>
+<html lang="en">
+<html>
+<head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width" />
+    <title>calendar-chloe</title>
+    <link rel="stylesheet" href="새거.css">
+    <link href="https://fonts.googleapis.com/css?family=Varela+Round&amp;display=swap" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css?family=Pacifico&amp;display=swap" rel="stylesheet">
+</head>
+
+<body>
+    <section></section>
+      <h1>calendar</h1>
+      <div class="calendar-container"></div>
+        <h2 id ="current-day"></h2> 
+        <h2 id ="current-date"></h2>  
+
```

Comment:
h2태그가 div 태그의 자식요소로 들어가지 않기 때문에 같은 인덴팅 라인으로 옮겨주세요.
```suggestion
      <div class="calendar-container"></div>
      <h2 id ="current-day"></h2> 
      <h2 id ="current-date"></h2>  
```